kalite/version.py

```diff
@@ -7,7 +7,7 @@
 # Must also be of the form N.N.N for internal use, where N is a non-negative integer
 MAJOR_VERSION = "0"
 MINOR_VERSION = "14"
-PATCH_VERSION = "0"
+PATCH_VERSION = "alpha3"
```

Comment:
This will also likely break syncing and other version-parsing dependent features.